### DISCUSSION TITLE
MonitorControl v4.1.0

### DISCUSSION THREAD

**Discussion by JoniVR** [MonitorControl/MonitorControl#1044] [Release Discussion]
### Improvements
* Updated menu bar icon and its behavior. by @JackHinkle in https://github.com/MonitorControl/MonitorControl/pull/880
* Onboarding Screen by @the0neyouseek in https://github.com/MonitorControl/MonitorControl/pull/906
* Fix Gamma Activity Enforcer window title typo by @robjwells in https://github.com/MonitorControl/MonitorControl/pull/952
* Improved dummy detection for newer versions of BetterDummy by @waydabber in https://github.com/MonitorControl/MonitorControl/pull/975

### Build related changes
* Use absolute paths for system framework references by @dimitarnestorov in https://github.com/MonitorControl/MonitorControl/pull/1012
* Fixed build scripts for m1 mac by @the0neyouseek in https://github.com/MonitorControl/MonitorControl/pull/1037
* Added check for M1 Pro/Max Macbook Pro HDMI by @the0neyouseek in https://github.com/MonitorControl/MonitorControl/pull/904

### Translations
* Added missing Hungarian translations and fixed some typos by @fejese in https://github.com/MonitorControl/MonitorControl/pull/905
* Added Portuguese (Brazil) translation - thanks to @gpnunes75 by @waydabber in https://github.com/MonitorControl/MonitorControl/pull/840
* Added Hungarian translations for onboarding + some minor translation fixes by @waydabber in https://github.com/MonitorControl/MonitorControl/pull/918
* German translations update by @jajoho in https://github.com/MonitorControl/MonitorControl/pull/935
* German translations for onboarding by @jajoho in https://github.com/MonitorControl/MonitorControl/pull/940
* Added Czech translation by @waydabber in https://github.com/MonitorControl/MonitorControl/pull/959
* Updated Chinese translations by @wjy2311077 in https://github.com/MonitorControl/MonitorControl/pull/1034
* Updated Taiwan Chinese translations by @kahotheo in https://github.com/MonitorControl/MonitorControl/pull/982 and https://github.com/MonitorControl/MonitorControl/pull/983
* Added Polish translations (thanks to @Zbynius) by @waydabber in https://github.com/MonitorControl/MonitorControl/pull/1027
* Improve Taiwan TC translations by @jrthsr700tmax in https://github.com/MonitorControl/MonitorControl/pull/1025
* Updated Dutch translations by @JoniVR in https://github.com/MonitorControl/MonitorControl/pull/1039

## New Contributors
* @fejese made their first contribution in https://github.com/MonitorControl/MonitorControl/pull/905
* @JackHinkle made their first contribution in https://github.com/MonitorControl/MonitorControl/pull/880
* @robjwells made their first contribution in https://github.com/MonitorControl/MonitorControl/pull/952
* @kahotheo made their first contribution in https://github.com/MonitorControl/MonitorControl/pull/983
* @jrthsr700tmax made their first contribution in https://github.com/MonitorControl/MonitorControl/pull/1025
* @dimitarnestorov made their first contribution in https://github.com/MonitorControl/MonitorControl/pull/1012
* @wjy2311077 made their first contribution in https://github.com/MonitorControl/MonitorControl/pull/1034

**Full Changelog**: https://github.com/MonitorControl/MonitorControl/compare/v4.0.2...v4.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/MonitorControl/MonitorControl/releases/tag/v4.1.0'>MonitorControl v4.1.0</a>.</em>

**Comment by bjarkebech** [comment]
This is a shining beacon of how a modern MacOS app should be. Works incredibly amazing. Thank you!!!!

**Comment by actuallyaridan** [comment]
Perfect application that does exactly what I need. Thank you!

**Comment by leokovaski** [comment]
Volume not working. LG Ultrawide Model 29UM69G

**Comment by alexTitakoff** [comment]
AWESOME!!

**Comment by brbetten** [comment]
Hi there, this app solved the lack of brightness controls for my LED cinema display! Thanks!  I was wondering for a complete uninstall though what Library files I should look for?

**Reply by the0neyouseek** [reply]
Thanks ! To completely uninstall you can delete these files / folder
- `~/Library/Containers/me.guillaumeb.MonitorControlHelper`
- `~/Library/Preferences/me.guillaumeb.MonitorControl.plist`

**Comment by banx78** [comment]
Thanks !! this is a real game changer, I was about to send myna monitor back !! Huge thanks !!

**Comment by sudianskyi** [comment]


**Reply by FAWN-MUSIC** [reply]
Same Here. Philips 32PFS4132/12
Macbook pro mid 2015 MacOS Big Sur 11.7.2

**Reply by olivierlord** [reply]
Volume does not work for me either...

**Reply by hardtimehotel** [reply]
Volume does not work on my Mac mini M2. Monitor AOC CU 34V5CW/BK is connected via HDMI to HDMI cable.
Brightness works oke.

**Reply by waydabber** [reply]
@hardtimehotel - take a look at this: https://github.com/MonitorControl/MonitorControl/discussions/750

**Comment by RadmirXAn** [comment]
Hi.
Volume not work if use HDMI to HDMI.
Volume work if use DISPLAY PORT to TYPE-C.
Brightness works in ALL.

**Comment by jayralbano** [comment]
Works well with my Mac Mini M1 with 2 monitors (LG and HP). Thank you!

**Comment by yoav-st** [comment]
Hi, I just installed the newest version and the controller are working but, when I set the brightness  to the max the led display doesn't rich the highest brightness that it have and the result are pretty dark.
![Screen Shot 2023-02-02 at 19 11 43](https://user-images.githubusercontent.com/123869533/216394398-41764f0d-220f-4726-bd9a-695f38725050.png)

**Reply by johandanielnilsson** [reply]
Hi yoav-st
Did you find a solution for this problem?
I ran into the same thing when I updated to 4.2

**Comment by jhay003** [comment]
thank you so much works perfectly.

**Comment by n0rt0nthec4t** [comment]
Is there going to be any newer binary builds released? Last one was 4.10 in may 2022!

**Reply by waydabber** [reply]
Hi @n0rt0nthec4t - we will see, it depends on contributions. I have an open PR to add support for M2 Pro Macs, it would make sense to produce a new binary with that change included + there are some translation updates as well. But aside from that, the current version seems fine. The team came to the conclusion that the original vision for MonitorControl is pretty much covered (or even exceeded in complexity) by the current feature set, adding more stuff would be counter-productive. But of course code contributions are still welcome! 

Note: you can try [BetterDisplay](https://betterdisplay.pro) as well if you want more features, that's a "no constraints" app where I implement every idea I have time for and seems at least remotely useful. :)

**Reply by n0rt0nthec4t** [reply]
@waydabber thanks. Be nice to have montiorcontrol updated for Sonoma. I like the simplicity of MonitorControl look vs BetterDisplay

**Reply by waydabber** [reply]
MonitorControl should work fine with Sonoma (I think).

I'd challange however the issue of "simplicity of MonitorControl vs BetterDisplay" as it comes up from time-to-time. :) BetterDisplay 2.x's SwiftUI rewrite looks rather nice I think and in terms of simplicity of menu presentation there's not much difference (obviously you can unfold BetterDisplay to be much more complex):

<img width="328" alt="Screenshot 2023-08-24 at 09 49 45" src="https://github.com/MonitorControl/MonitorControl/assets/37590873/d3373e60-6675-4d9a-a964-1a6f945ec182">

<img width="266" alt="Screenshot 2023-08-24 at 09 50 00" src="https://github.com/MonitorControl/MonitorControl/assets/37590873/27871161-0cf2-4890-867e-9ed5d0dda7cb">

But for sure this is a matter of preference, I like them both (since MonitorControl's current UI is also mostly my work, it would be strange of me to say it's not nice. :)).

**Reply by n0rt0nthec4t** [reply]
Yes, definitely subjective view :-)  Appreciate your work on both apps

<img width="279" alt="Screenshot 2023-08-25 at 8 46 25 am" src="https://github.com/MonitorControl/MonitorControl/assets/19759130/f41fd512-3770-4421-872d-176ce7724613">
<img width="284" alt="Screenshot 2023-08-25 at 8 46 35 am" src="https://github.com/MonitorControl/MonitorControl/assets/19759130/7f480a76-967f-42a7-9fd6-5e8b97d772f0">
